axelar-chains-config/info/mainnet.json

```diff
@@ -1469,7 +1480,8 @@
       "gasOptions": {
         "gasLimit": 8000000,
         "gasPriceAdjustment": 2
-      }
+      },
+      "finality": 60
```

Comment:
where are you getting this number from? check the chains config for the setting if the chain doesn't finalized tag support

Reply:
this is the minutes to wait for block to be confirmed if finality tag is not supported.

Reply:
and the values for all have been taken from chains config.

Reply:
Have changed it number of blocks to wait for, should I generate the value by script or replace with the ones in chains config?

Reply:
we can't programmatically determine block confirmations, since it comes from reading their docs and setting it in chain configs, not via RPC

Reply:
there should be a separate `approxFinalityWaitTime` to indicate the time taken on avg to finality